holobot/sdk/utils/list_utils.py

```diff
@@ -2,13 +2,8 @@
 
 T = TypeVar("T")
 
-def pad_left(list: list[T], value: T, length: int) -> list[T]:
+def pad_left(list_: list[T], value: T, length: int) -> list[T]:
```

Comment:
Shadowing `list`

Reply:
Personally, I'm not a fan of pre-/suffixing variable names. What about calling it `source` instead?

Reply:
Sure.